library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-            *** Fragmented-discharge-pattern
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** Discharge-pattern <nowiki>[Describes the organization of the EEG signal within the discharge (distinguish between single and repetitive discharges). (Source: Beniczky ea 2017, Table 6;  Beniczky ea 2013, Appendix 4.)]</nowiki>
+            *** Single-discharge-pattern <nowiki> {suggestedTag=Finding-incidence} [Applies to the intra-burst pattern: a graphoelement that is not repetitive; before and after the graphoelement one can distinguish the background activity. (Source: Beniczky ea 2017, Table 6;  Beniczky ea 2013, Appendix 4.)]</nowiki>
+            *** Rhythmic-trains-or-bursts-discharge-pattern <nowiki> {suggestedTag=Finding-prevalence, suggestedTag=Finding-frequency} [Applies to the intra-burst pattern: a non-ictal graphoelement that repeats itself without returning to the background activity between them. The graphoelements within this repetition occur at approximately constant period. (Source: Beniczky ea 2017, Table 6;  Beniczky ea 2013, Appendix 4.)]</nowiki>
+            *** Arrhythmic-trains-or-bursts-discharge-pattern <nowiki> {suggestedTag=Finding-prevalence} [Applies to the intra-burst pattern: a non-ictal graphoelement that repeats itself without returning to the background activity between them. The graphoelements within this repetition occur at inconstant period. (Source: Beniczky ea 2017, Table 6;  Beniczky ea 2013, Appendix 4.)]</nowiki>
+            *** Fragmented-discharge-pattern <nowiki> [Source: Beniczky ea 2017, Table 6.] </nowiki>
 
-        ** Periodic-discharge-time-related-features <nowiki>[Periodic discharges not further specified (PDs) time-relayed features tags.]</nowiki>
-            *** Periodic-discharge-duration {suggestedTag=Property-not-possible-to-determine}
-                **** Very-brief-periodic-discharge-duration <nowiki>[Less than 10 sec.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Brief-periodic-discharge-duration <nowiki>[10 to 59 sec.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Intermediate-periodic-discharge-duration <nowiki>[1 to 4.9 min.]</nowiki>
+        ** RPP-time-related-features <nowiki>[Time related features for Rhythmic or Periodic Patterns in critically ill patients (RPPs). (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+            *** RPP-duration <nowiki> {suggestedTag=Property-not-possible-to-determine} [Duration (for PDs, RDA and SW). (Source: Beniczky ea 2017, Table 8.)]</nowiki>
```

Comment:
I don't think that Property-not-possible-to-determine"  should be a suggested tag maybe suggest in the description that this tag can be used if the property cannot be determined.

Reply:
done